packages/remix-dev/cli/migrate/migrations/convert-to-javascript/index.ts

```diff
@@ -10,16 +10,17 @@ import { convertTSFilesToJS } from "./convertTSFilesToJS";
 
 const TRANSFORM_PATH = join(__dirname, "transform");
 
-export const convertToJavaScript: MigrationFunction = async ({
-  flags,
+export const convertToJavaScript: MigrationFunction = async (
   projectDir,
-}) => {
+  flags = {}
+) => {
   let config = await readConfig(projectDir);
+  let isInteractive = require.main === module;
```

Comment:
@pcattori This seems to break our test 🤔

https://github.com/remix-run/remix/runs/7009823608?check_suite_focus=true#step:6:163

Reply:
Tests still seem to fail 😕

https://github.com/remix-run/remix/runs/7011654283?check_suite_focus=true#step:6:190

Reply:
Are the tests passing for you locally?

Reply:
No they're failing locally as well with this 😕

Reply:
If I run `yarn jest __tests__/replace-remix-imports-test.ts` within `packages/remix-dev`, the test passes, but running `yarn jest` (either in `packages/remix-dev` or in repo root) causes the test to fail.